docker/lr-trgt/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM us.gcr.io/broad-dsp-lrma/lr-pb:0.1.40
```

Comment:
Is there a reason you'd like to base your image on this particular one?
We are trying to avoid interdependencies between dockers in this repo (this way, one has less to worry about when updating one image).

Here's what I'd propose:

- use a minimal base image (ubuntu, or python, or whichever appropriate)
- install dependencies required by `trgt` explicitly in this docker file.

Reply:
@SHuang-Broad, thanks so much for all this excellent feedback!

No, I didn't have a great reason for branching this image off of lr-pb. I have switched it now to ubuntu:20.04.